labs/3L.md

```diff
@@ -0,0 +1,108 @@
+
+# Auto-grader Instructions
+
+## Update Auto-grader
+First, update the auto-grader by running the following command:
+```bash
+cargo install --git=https://github.com/DhruvDh/umm.git
+```
+
+## Opening Project in IDE
+Project files: [3L_test-driven-development.zip](<Link_todo>)
+
+1. Unzip the files preferably into an `ITSC 2214` folder for this class.
```

Comment:
make it more clear that 2214 folder is a parent folder for all labs. inside we have individual folders for each lab